index.html

```diff
@@ -85,6 +85,268 @@ <h1>
               </div>
             </section>
           </div>
+
+        <div class="all_posts"> 
+          <section class="posts">
+            <div class="contaier posts_container">
+              <article class="post">
+                <div class="image">
+                  <img src="images/Pics.png" alt="">
+                </div>
+                <div class="info">
+                  <h4>Tonic</h4>
+                  <ul>
+                    <li class="canopy">CANOPY</li>
+                    <li class="dot"><img src="/images/Dot.png" alt=""></li>
+                    <li class="developer">Back End Dev</li>
+                    <li class="dot"><img src="/images/Dot.png" alt=""></li>
+                    <li class="year">2015</li>
+                  </ul>
+                  <p>A daily selection of privately<br>
+                     personalized reads; no accounts or<br>
+                     sign-ups required.
+                  </p>
+                  <a href="html" class="btn">HTML</a>
+                  <a href="css" class="btn">CSS</a>
+                  <a href="JavaScript" class="btn">JavaScript</a>
```

Comment:
Consider wrapping the languages used in a `<ul>` tag and each language in a `<li>` tag. Do the same for all the project cards.